code/modules/cm_marines/equipment/gear.dm

```diff
@@ -147,27 +147,27 @@
 	desc = "<B>Instructions:</B> Extract food using maximum firepower. Eat.\n\nOn the box is a picture of a shouting Squad Leader. \n\"YOU WILL EAT YOUR NUTRIENT GOO AND YOU WILL ENJOY IT, MAGGOT.\""
 	icon_state = "mre1"
 
-	New()
-		..()
-		pixel_y = rand(-3,3)
-		pixel_x = rand(-3,3)
-		for(var/i = 0,i < 6,i++)
-			var/rand_type = rand(0,8)
-			switch(rand_type)
-				if(0 to 2)
-					new /obj/item/reagent_container/food/snacks/protein_pack(src)
-				if(3)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal1(src)
-				if(4)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal2(src)
-				if(5)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal3(src)
-				if(6)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal4(src)
-				if(7)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal5(src)
-				if(8)
-					new /obj/item/reagent_container/food/snacks/mre_pack/meal6(src)
+/obj/item/storage/box/uscm_mre/Initialize(mapload, ...)
+	. = ..()
+	pixel_y = rand(-3,3)
+	pixel_x = rand(-3,3)
+	for(var/i = 0,i < 6,i++)
+		var/rand_type = rand(0,8)
+		switch(rand_type)
+			if(0 to 2)
```

Comment:
I know this is just changing the indentation but god, this could use to be a bit prettier